### ISSUE TITLE
Page Editor error banner showing stringified object on request error

### ISSUE THREAD

**Issue by twschiller** [pixiebrix/pixiebrix-extension#3131]
![image](https://user-images.githubusercontent.com/1879821/162578507-b0624c76-ce0b-423e-b428-e78876388454.png)

**Comment by fregante** [comment]
> Might be resolved by #3126

Yeah I really hope we don't need to handle RTK error objects separately everywhere
